airflow/contrib/hooks/jenkins_hook.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from airflow.hooks.base_hook import BaseHook
+
+import jenkins
+import distutils
+
+from airflow.utils.log.LoggingMixin import LoggingMixin
+log = LoggingMixin().logger
+
+class JenkinsHook(BaseHook):
+    """
+    Hook to manage connection to jenkins server
+    """
+
+    def __init__(self, conn_id='jenkins_default'):
+        connection = self.get_connection(conn_id)
+        self.connection = connection
+        connectionPrefix = 'http'
+        if connection.extra == None or connection.extra == '':
+            conneciton.extra = 'false'
+        if distutils.util.strtobool(connection.extra):
```

Comment:
What do you do here?

Reply:
I had a comment, but I was setting a default value in connection.extra to avoid having a ValueError thrown when I do the strtobool. 
The connection.extra field host the value to tell if we should use http or https (the label has been renamed in the connection form to something understandable).